examples/basic/variables.tf

```diff
@@ -0,0 +1,94 @@
+variable "pvs_zone" {
+  description = "IBM Cloud PVS Zone. Valid values: sao01,osa21,tor01,us-south,dal12,us-east,tok04,lon04,lon06,eu-de-1,eu-de-2,syd04,syd05"
```

Comment:
We can add a validation block to check the value here is in the list

Reply:
I think we should also use full names whenever possible - eg: IBM Cloud PVS -> IBM Cloud PowerVS (in the description)